src/dmd/backend/outbuf.d

```diff
@@ -52,10 +54,28 @@ struct Outbuffer
     void enlarge(uint nbytes);
 
     // Write n zeros; return pointer to start of zeros
-    void *writezeros(size_t n);
+    void *writezeros(uint n)
```

Comment:
Is the `size_t` => `uint` change on purpose ?

Reply:
Yes. Miserable mangling problems with size_t, that won't go away without upgrading the boot compiler or getting it all in D.